pkg/api/dashboard.go

```diff
@@ -840,7 +840,10 @@ func (hs *HTTPServer) ValidateDashboard(c *contextmodel.ReqContext) response.Res
 	// work), or if schemaVersion is absent (which will happen once the Thema
 	// schema becomes canonical).
 	if err != nil || schemaVersion >= dashboard.HandoffSchemaVersion {
-		_, _, validationErr := dk.JSONValueMux(dashboardBytes)
+		// Schemas expect the dashboard to live in the spec field
+		k8sResource := `{"spec": ` + cmd.Dashboard + "}"
```

Comment:
this looks like the one place that tries to validate anything

Reply:
yep, this should be the only one, since we don't have the devenv validation anymore